### DISCUSSION TITLE
Game renders in a small corner

### DISCUSSION THREAD

**Discussion by attestwire** [elishacloud/dxwrapper#317] [General]
Hi there Elisha, @elishacloud 

Thanks so much for your work! I'm trying to play Arcanum on Windows 10 on an intel mac, native resolution is 3070x1920. When I use anything less than that I just get a small corner of the screen rendering

Here's an example of what I'm seeing --

[Reddit screenshot](https://www.reddit.com/media?url=https%3A%2F%2Fpreview.redd.it%2Ffull-screen-issues-on-windows-10-v0-3jzdmaelypzd1.jpeg%3Fwidth%3D1080%26crop%3Dsmart%26auto%3Dwebp%26s%3D8327bc3bfc30d0362df533e90a265a40503060e4)

Any ideas what might be causing this? Again it works perfectly at full native resolution, but if I want to scale down to something like 720p then it just stays in the corner and is too small to play really.

Your help is greatly appreciated!

**Comment by elishacloud** [comment]
What settings do you have enabled in dxwrapper?  Also, where you are setting the resolution to 3070x1920?

You could try and enable the dxwrapper `FullscreenWindowMode` option.

**Reply by attestwire** [reply]
Here are the settings. I'm using a hi res mod that uses DX wrapper from what I can tell. There's a setting in the hi res mod config file for the resolution. 

`;; Config file for DirectX DLL Wrapper
;;
;; For details, see: https://github.com/elishacloud/dxwrapper/wiki/Configuration
;;
[General]
RealDllPath                = AUTO
WrapperMode                = AUTO
LoadCustomDllPath          = 
ExcludeProcess             = 
IncludeProcess             = 
RunProcess                 = 
WaitForProcess             = 0
DisableLogging             = 0

[Plugins]
LoadPlugins                = 0
LoadFromScriptsOnly        = 0

[Compatibility]
Dd7to9                     = 0
D3d8to9                    = 0
DDrawCompat                = 0
Dinputto8                  = 0
DisableGameUX              = 0
DxWnd                      = 0
EnableDdrawWrapper         = 0
EnableD3d9Wrapper          = 0
EnableDinput8Wrapper       = 0
EnableDsoundWrapper        = 0
HandleExceptions           = 0
SingleProcAffinity         = 0

[DDrawCompat]
DDrawCompat20              = 0
DDrawCompat21              = 0
DDrawCompat31              = 0
DDrawCompatDisableGDIHook  = 0
DDrawCompatNoProcAffinity  = 0

[ddraw]
ConvertToDirectDraw7       = 0
ConvertToDirect3D7         = 0
DdrawFixByteAlignment      = 0
DdrawOverrideBitMode       = 0

[Dd7to9]
AutoFrameSkip              = 0
DdrawEmulateSurface        = 0
DdrawRemoveScanlines       = 0
DdrawReadFromGDI           = 0
DdrawWriteToGDI            = 0
DdrawLimitDisplayModeCount = 0
DdrawUseNativeResolution   = 0
DdrawClippedWidth          = 0
DdrawClippedHeight         = 0
DdrawOverrideWidth         = 0
DdrawOverrideHeight        = 0
DdrawOverrideRefreshRate   = 0
DdrawIntegerScalingClamp   = 0
DdrawMaintainAspectRatio   = 0

[d3d9]
AnisotropicFiltering       = 0
AntiAliasing               = 0
EnableVSync                = 0
EnableWindowMode           = 0
ForceMixedVertexProcessing = 0
ForceSystemMemVertexCache  = 0
ForceVsyncMode             = 0
FullscreenWindowMode       = 0
WindowModeBorder           = 0

[FullScreen]
FullScreen                 = 0
ForceWindowResize          = 0
WaitForWindowChanges       = 0

[dinput8]
FilterNonActiveInput       = 0

[dsound]
Num2DBuffers               = 0
Num3DBuffers               = 0
ForceCertification         = 0
ForceExclusiveMode         = 0
ForceSoftwareMixing        = 0
ForceHardwareMixing        = 0
ForceHQ3DSoftMixing        = 0
ForceNonStaticBuffers      = 0
ForceVoiceManagement       = 0
ForcePrimaryBufferFormat   = 0
PrimaryBufferBits          = 16
PrimaryBufferSamples       = 44100
PrimaryBufferChannels      = 2
AudioClipDetection         = 0
`

**Reply by elishacloud** [reply]
Looks like you are not using any dxwrapper settings.  Try enabling the `Dd7to9` option.
